azure-iot-hub-devicesdk/samples/send_telemetry_device.py

```diff
@@ -3,48 +3,59 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 import os
-import time
-from azure.iot.hub.devicesdk.device_client import DeviceClient
-from azure.iot.hub.devicesdk.auth.authentication_provider_factory import from_connection_string
-from azure.iot.hub.devicesdk.message import Message
+import asyncio
+import logging
 import uuid
+from azure.iot.hub.devicesdk import DeviceClient, Message
+from azure.iot.hub.devicesdk.auth.authentication_provider_factory import (
+    from_connection_string,
+)  # this is a overlong import, fix
```

Comment:
># this is a overlong import, fix [](start = 3, length = 32)

I agree 100% as you've seen in my other comments.  We should open a TFS item to make sure we do this.